woocommerce/woocommerce
# GDPR - log audit

This PR addresses PII stored in debug logs.

1. Webhook body is no longer logged unless DEBUG mode is enabled. No other PII data is included.
2. PayPal response data is no longer stored to meta.
3. PayPal logs mask user address data and no longer log full request/responses which could include PII
4. The debug setting in paypal includes a disclaimer about clearing logs when done.
5. When debug in paypal is disabled, logs are cleared. I’ve added support for the DB logger method too to allow clearing by source.

To test:

- Enable some webhooks and trigger them
- Checkout using PayPal stadard.
- Check WC > Status > Logs and ensure data is masked or missing.
